external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -44,7 +44,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
 
     <!-- Test Data -->
-    <PackageReference Include="System.IO.Compression.TestData" Version="1.0.6-prerelease" />
+    <PackageReference Include="System.IO.Compression.TestData" Version="1.0.7-prerelease" />
```

Comment:
@ViktorHofer why was this version bump necessary?

Reply:
I uploaded the self-compiled test data packages to the feed. As some tests started to fail it was clear to me that someone must have updated a package without checking the change in. So I had to re-upload that package and I used the one that was still in my nuget cache. With my recent changes in corefx-testdata I made sure that all the testdata project are now correct.

Reply:
@ViktorHofer the issue was that the 1.0.6 package (that you pushed to blob storage) had Windows line endings, while the one in myget had Unix line endings. It looks like the tests expect the compressed test data to match Unix line endings. I've re-uploaded a 1.0.6 w/ Unix line endings to blob storage, so that rel/3.0 will start passing again. We should make sure that in the future we always build the package w/ Unix line endings - I'm playing around with `.gitattributes` file now.

Reply:
I see, @ericstj and @wtgodbe just figured it out why, it is because of the line endings, the 1.0.6 version you pushed has Windows Line endings and the 1.0.7 has linux line endings. This broke release/3.0 branch because it depends on the 1.0.6 package and the dotnet-blob feed is first in the list of sources.

Reply:
ok I see, I hit the same issue some hours ago locally and had to change my gitattributes. sorry that I caused issues here, that wasn't my intention.

Reply:
No worries